homeassistant/components/sensor/dsmr.py

```diff
@@ -267,6 +272,11 @@ def state(self):
         if self._obis == obis.ELECTRICITY_ACTIVE_TARIFF:
             return self.translate_tariff(value)
 
+        try:
+            value = round(float(value), self._config[CONF_PRECISION])
+        except TypeError:
+            value = value
```

Comment:
If we will keep the try/except we can just `pass` here.

Reply:
I've changed it and commited it.